docs: Add migration documentation from 1.0 to 1.1 (#695)
unoplatform/Uno.Themes: README.md

| CheckBox         | MaterialCheckBoxStyle <br> MaterialSecondaryCheckBoxStyle                         |
| ComboBox         | MaterialComboBoxStyle                                                             |
| CommandBar       | MaterialCommandBarStyle                                                           |

> | ListView         | MaterialListViewStyle <br> |

Reviewer: ```suggestion
| ListView         | MaterialListViewStyle <br> MaterialListViewDetailsStyle |
```

> | ListViewItem     | MaterialListViewItemStyle <br> |

Reviewer: ```suggestion
| ListViewItem     | MaterialListViewItemStyle |
```

</resources>
```

## Migration

### 1.0 to 1.1

- Color Palette Override

	Now you have the possibility to override the Material color palette with your own color palette. See the #Getting Started section for more details.
	```xml
	<MaterialColors xmlns="using:Uno.Material"
					OverrideSource="ms-appx:///ColorPaletteOverride.xaml" />
	```

- Namespace breaking changes

> 	BREAKING CHANGE: Everything (controls, extensions, converters,...) is now under `Uno.Material` and under `Uno.Cupertino`.

Reviewer: ```suggestion
	BREAKING CHANGE: Everything (controls, extensions, converters, ...), previously under `Uno.Material.*` or `Uno.Cupertino.*`, has now been moved under `Uno.Material` or `Uno.Cupertino`.
```

	| Divider                   | MaterialDividerStyle |

- Some controls have been removed

	List of the controls and styles that have been removed from Uno.Themes:

	| **Controls**              | **StyleNames**                                                                |
	|---------------------------|------------------------------------------------------------------------------|
	| BottomNavigationBar       | MaterialBottomNavigationBarStyle                                              |
	| ExpandingBottomSheet      | MaterialExpandingBottomSheetStyle                                             |
	| ModalStandardBottomSheet  | MaterialModalStandardBottomSheetStyle                                         |
	| StandardBottomSheet       | MaterialStandardBottomSheetStyle                                              |
	| SnackBar                  | MaterialSnackBarStyle                                                         |

	- BottomNavigationBar was replaced by TabBar in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but it is not an exact 1:1 replacement.

> 	In the meant time, if you really need the badge and/or other customizabilities, you have two options:

Reviewer 2: ```suggestion
	In the mean time, if you really need the badge and/or other customizability, two options are available:
```

	Import locally the old sources (control + style) from Uno.Themes; 

		OR

		Copy the MaterialBottomTabBarItemStyle from [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), and modify the style to suit your needs.

> 		(Note that there is two copy of it, one for ios and android and one for rest of the platforms: uwp, skia, wasm, etc...)

Reviewer 2: What's "it"? Avoid pronouns if possible.

> 		(Note that there is two copy of it, one for ios and android and one for rest of the platforms: uwp, skia, wasm, etc...)

Reviewer 3: ```suggestion
		(Note that there are two copies of the style, one for iOS and Android and one for rest of the platforms: UWP, Skia, WASM, etc...)
```

	- For StandardBottomSheet and ModalStandardBottomSheet
	It's replaced by using a Flyout and the MaterialFlyoutPresenterStyle that is allowing you to have a bottom sheet.

		For example:
		```xml
		<Flyout Placement="Full"
                LightDismissOverlayMode="On"
                FlyoutPresenterStyle="{StaticResource MaterialFlyoutPresenterStyle}">
			<Grid MaxHeight="370"
				  VerticalAlignment="Bottom">
				...Your bottom sheet content...
			</Grid>
		</Flyout>
		```

> 	- For SnackBar, at the moment we didn't add yet in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui) a control that will replace SnackBar but it's planned.

Reviewer 2: ```suggestion
	- SnackBar is not yet added in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but a control that will replace SnackBar is planned.
```

> 	- For SnackBar, at the moment we didn't add yet in [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui) a control that will replace SnackBar but it's planned.

Reviewer 3: ```suggestion
	- For SnackBar: so far no replacement for SnackBar has been added to [Uno.Toolkit.UI](https://github.com/unoplatform/uno.toolkit.ui), but it's planned to add one in a future version.
```
